### PR TITLE
[DR-1594] Updates on cdn image to work with deployment scripts

### PR DESCRIPTION
**Pull request by gpgpublickey**
## Background

Some updates in the dockerfile and cdn-uploader.js to integrate with build docker image and the deployment bash script.
  
## Changes done

Dockerfile source of builds, add local envs
Updates on cdn-uploader js
Update readme

## Pending to be done
  
N/A
  
## Notes
  
Environments variables defined with ENV in the dockerfile will be overrided by the host machine defined environment variables. This will be added in the README file.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,7 +7,6 @@ COPY cdn-uploader.js .
 RUN npm init --yes
 RUN npm install basic-ftp
 RUN mkdir /source
-VOLUME /source
```
Comment (andresmoschini) [inline Dockerfile]: Why did you remove it? I would like to declare the volume here in the same way that we are declaring the environment variables to document what we need to configure to _run_ this image.

**Comment by andresmoschini** [comment]
We have talked with @cristiangiagante about simplifying readme removing usage details.

**Comment by gpgpublickey** [comment]
@andresmoschini @mvillaseco I update the readme to simplify the content.

File: README.md [hunk 2]
```diff
@@ -9,88 +9,38 @@ Docker HUB url: https://hub.docker.com/r/dopplerrelay/doppler-relay-akamai-publi
...
-export AKAMAI_CDN_HOSTNAME=example.cdn.com # CDN netstorage domain
-export AKAMAI_CDN_USERNAME=example         # Username from Akamai CDN
-export AKAMAI_CDN_PASSWORD=12345           # Password from Akamai CDN
-export AKAMAI_CDN_CPCODE=56789             # Cpcode provided by Akamai Netstorage, this is the root folder with write privileges inside the Akamai CDN
-```
-
-First Step, set the docker-compose.yml:
-
-The environment variables in the host machine are integrated in the docker image by the docker-compose.yml file. It is required by the cdn-uploader.js script.
+You need to define this environment variables. They are needed to login with the CDN ftp and cdn uploader script.
```
Comment (andresmoschini) [inline README.md]: I do not need to define any environment variable to run it :P

File: README.md [hunk 3]
```diff
@@ -9,88 +9,38 @@ Docker HUB url: https://hub.docker.com/r/dopplerrelay/doppler-relay-akamai-publi
...
-WORKDIR .
-COPY --from=mseditor-env /refactor/builds /builds
-CMD ["node", "cdn-uploader.js", "mseditor", "builds"]
+docker run --rm \
+	-e AKAMAI_CDN_HOSTNAME \
+	-e AKAMAI_CDN_USERNAME \
+	-e AKAMAI_CDN_PASSWORD \
+	-e AKAMAI_CDN_CPCODE \
+	-e "PROJECT_NAME=mseditor" \
+	-e "VERSION_NAME=$1" \
```
Comment (andresmoschini) [inline README.md]: $1 has no value, I think that for this example you should hardcode the version

Comment (gpgpublickey) [reply README.md]: Perfect, i will do it.

File: README.md [hunk 4]
```diff
@@ -9,88 +9,38 @@ Docker HUB url: https://hub.docker.com/r/dopplerrelay/doppler-relay-akamai-publi
...
+VERSION_NAME=v1.0.0-build1234       # Change this by your build identifier, this can be overrided dinamically using a bash script ex: docker run -e "VERSION_NAME=$1" this will receive $1 like a bash command line argument.
 ```
 
-Second Step, define your dockerfile:
-
-In order to implement the image you need to call it from a Dockerfile, you can multistage it with another one if you want.
+IMPORTANT: Environments variables defined with ENV in the dockerfile will be overrided by the host machine defined environment variables. This will be added in the README file.
 
-Example using multistage:
+## First Step, run it with bash script: ##
```
Comment (andresmoschini) [inline README.md]: It is not a bash script, it is simply a command

File: README.md [hunk 5]
```diff
@@ -9,88 +9,38 @@ Docker HUB url: https://hub.docker.com/r/dopplerrelay/doppler-relay-akamai-publi
...
-      dockerfile: editor.Dockerfile # Name of your dockerfile implementing doppler-docker-akamai-publish image
-    environment: 
-      - 'AKAMAI_CDN_HOSTNAME=${AKAMAI_CDN_HOSTNAME}'
-      - 'AKAMAI_CDN_USERNAME=${AKAMAI_CDN_USERNAME}'
-      - 'AKAMAI_CDN_PASSWORD=${AKAMAI_CDN_PASSWORD}'
-      - 'AKAMAI_CDN_CPCODE=${AKAMAI_CDN_CPCODE}'
+AKAMAI_CDN_HOSTNAME=example.cdn.com # CDN netstorage domain
+AKAMAI_CDN_USERNAME=example         # Username from Akamai CDN
+AKAMAI_CDN_PASSWORD=12345           # Password from Akamai CDN
+AKAMAI_CDN_CPCODE=56789             # Cpcode provided by Akamai Netstorage
```
Comment (andresmoschini) [inline README.md]: I think that it is not necessary to define any variable, but at least these are being used in your example calling _docker run_, but you are not using _PROJECT_NAME_ and neither _VERSION_NAME_

Comment (gpgpublickey) [reply README.md]: ready!

File: README.md [hunk 6]
```diff
@@ -9,88 +9,25 @@ Docker HUB url: https://hub.docker.com/r/dopplerrelay/doppler-relay-akamai-publi
 
 This image is ready to upload to akamai CDN the resource files. Contains the cdn-uploader script and his dependencies ready to run.
 
-Zero Step, define your environment variables:
+IMPORTANT: Environments variables defined with ENV in the dockerfile will be overrided by the host machine defined environment variables. This will be added in the README file.
```
Comment (andresmoschini) [inline README.md]: I think that this paragraph is not correct, because it depends on how it is implemented in host machine.

Maybe it is better to indicate that it is necessary to pass/define _AKAMAI_CDN_HOSTNAME_, _AKAMAI_CDN_USERNAME_, _AKAMAI_CDN_PASSWORD_, _AKAMAI_CDN_CPCODE_, _PROJECT_NAME_, _VERSION_NAME_ environment variables and _/source_ volume.

**Comment by andresmoschini** [comment]
👍
